governance/xc_admin/packages/crank_executor/src/index.ts

```diff
@@ -213,5 +213,9 @@ async function run() {
 }
 
 (async () => {
-  await run();
+  try {
+    await run();
+  } catch (err) {
+    console.error(err);
+  }
```

Comment:
this no longer throws on errors -- is that intentional?

Reply:
i mean it's not just improving logging, it's also suppressing errors

Reply:
I'll make it throw...

Reply:
Without this trick the error wasn't getting printed